spring-batch/src/main/java/org/baeldung/conditionalflow/config/NumberInfoConfig.java

```diff
@@ -0,0 +1,85 @@
+package org.baeldung.conditionalflow.config;
+
+import org.baeldung.conditionalflow.NumberInfoDecider;
+import org.baeldung.conditionalflow.model.NumberInfo;
+import org.baeldung.conditionalflow.step.*;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableBatchProcessing
```

Comment:
Again the formatting is off on this class. Let's fix that please.

Reply:
Done